pylint/checkers/typecheck.py

```diff
@@ -985,6 +985,8 @@ def visit_attribute(self, node):
                     continue
             except AttributeError:
                 continue
+            except astroid.DuplicateBasesError:
```

Comment:
```suggestion
            except astroid.DuplicateBasesError:
            # See https://github.com/PyCQA/pylint/pull/4317
```

Reply:
I'm not sure this is worth it. For one thing, you can get this information via git blame. Additionally the added test will catch if it's missing.